smartsheet-csharp-sdk/main/Smartsheet/Api/HomeResources.cs

```diff
@@ -67,7 +67,7 @@ public interface HomeResources
         /// <exception cref="SmartsheetException">if any other error occurred during the operation</exception>
         /// <returns> the resource (note that if there is no such resource, this method will throw ResourceNotFoundException
         /// rather than returning null). </returns>
-        PersonalFolder GetFoldersPersonal(IEnumerable<HomeInclusion> includes, IEnumerable<HomeExclusion> excludes);
+        PersonalFolder GetFoldersPersonal(IEnumerable<HomeInclusion>? includes = null, IEnumerable<HomeExclusion>? excludes= null);
```

Comment:
nitpick: spacing
```suggestion
        PersonalFolder GetFoldersPersonal(IEnumerable<HomeInclusion>? includes = null, IEnumerable<HomeExclusion>? excludes = null);
```
#

Reply:
Fixed it up in the latest commit thank you!